http-api/src/lib.rs

```diff
@@ -211,22 +281,19 @@ async fn periodic(ctx: Context, interval: time::Duration) {
     }
 }
 
-/// Return the peer id for the node identity.
-/// `GET /v1/peer`
-async fn peer_handler(peer_id: PeerId) -> Result<impl warp::Reply, warp::Rejection> {
-    let response = json!({
-        "id": peer_id.to_string(),
-    });
-    Ok(warp::reply::json(&response))
-}
-
 async fn recover(err: Rejection) -> Result<impl Reply, std::convert::Infallible> {
     let (status, msg) = if err.is_not_found() {
         (StatusCode::NOT_FOUND, None)
     } else if let Some(Error::NotFound) = err.find::<Error>() {
         (StatusCode::NOT_FOUND, None)
     } else if let Some(Error::NoHead(msg)) = err.find::<Error>() {
         (StatusCode::NOT_FOUND, Some(*msg))
+    } else if let Some(Error::Auth(msg)) = err.find::<Error>() {
+        (StatusCode::BAD_REQUEST, Some(*msg))
+    } else if let Some(Error::SiweParseError(_)) = err.find::<Error>() {
+        (StatusCode::BAD_REQUEST, None)
+    } else if let Some(Error::SiweVerificationError(_)) = err.find::<Error>() {
+        (StatusCode::BAD_REQUEST, None)
```

Comment:
@cloudhead I was not entirely sure if we said to leave the Siwe Errors as `BAD_REQUEST` or to change some to `UNAUTHORIZED` IMO I think both are  the result of a bad request..